lightningd/peer_comms.c

```diff
@@ -0,0 +1,20 @@
+#include <lightningd/peer_comms.h>
+#include <unistd.h>
+
+static void destroy_peer_comms(struct peer_comms *pcomms)
+{
+	if (pcomms->peer_fd != -1)
+		close(pcomms->peer_fd);
+	if (pcomms->gossip_fd != -1)
+		close(pcomms->gossip_fd);
+	if (pcomms->gossip_store_fd != -1)
+		close(pcomms->gossip_store_fd);
+}
+
+struct peer_comms *new_peer_comms(const tal_t *ctx)
```

Comment:
here should we initialize `fd`s with -1 for safty?

Reply:
I'd be tempted to, but we're literally freeing the memory at the end of this call.